dash-renderer/src/components/error/FrontEnd/FrontEndError.react.js

```diff
@@ -108,7 +108,9 @@ function UnconnectedErrorContent({error, base}) {
                 </div>
             )}
             {/* Backend Error */}
-            {typeof error.html !== 'string' ? null : (
+            {typeof error.html !== 'string' ? null : error.html.indexOf(
+                  '<!DOCTYPE HTML'
+              ) !== -1 ? (
```

Comment:
```suggestion
              ) === 0 ? (
```
Just in case this were to show up somewhere in the error message, only look at the beginning of the string.